packages/orm/src/reactiveHints.ts

```diff
@@ -120,25 +120,40 @@ export function reverseReactiveHint<T extends Entity>(
             },
           );
         }
-        case "m2m":
+        case "m2m": {
```

Comment:
Reversals of m2m is different enough now (it already had some special casing on the old line 134) that I broke it out into its own case